src/Router/Configure/Routes.php

```diff
@@ -89,17 +92,22 @@ private function addRoute(
         string $action = '__invoke',
         ?string $pathPattern = null,
     ): void {
+        $path = ($path === '/') ? '' : $path;
+
+        $isPathValid = (new PathValidator())($path);
```

Comment:
Wouldn't be simpler to read something like `PathValidator::isPathValid($path)`? Or similar. I mean to use a static method instead of "invoking a new instance" on the fly. Because what we are doing here is to create a higher dependency from the `PathValidator` to the `Routes` class, which is practically the same as if we would create a global/static function. What do you think, @ynnoig?

Reply:
I would go for the `PathValidator::isValid($path)` as the `::isValid()` provides more context than the invoke method in this case 👍🏼